.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+*.
```

Comment:
any reason we are ignoring  *. files ?

Reply:
I was going to ignore .env files because i read an article that we should not make our env files public because it contains sensitive information.
[](https://salferrarello.com/add-env-to-gitignore/)

Reply:
> I was going to ignore .env files because i read an article that we should not make our env files public because it contains sensitive information. [https://salferrarello.com/add-env-to-gitignore/](url)

ok,

& we should try using [this](https://github.com/JavascriptShinobis/messaging-app/settings/secrets/actions) for secrets